README.md

```diff
@@ -1829,7 +1832,7 @@ trapped many employees and
 [bankrupted people](http://blog.sfgate.com/dgreenberg/2012/06/22/tax-advice-from-the-dot-com-bubble-beware-of-isos/)
 during past dot-com busts.
 Now more people know about it, but it’s still a significant obstacle to plan around.
-(Note that if your AMT is for events prior to 2008,
+(Note that if your AMT applies to events prior to 2008,
```

Comment:
Remove parens and move to a standalone own paragraph? This really is just a separate point.

Reply:
Fixed.